configuration.go

```diff
@@ -9,16 +9,22 @@ import (
 
 // DefaultCache the struct
 type DefaultCache struct {
-	AllowedHTTPVerbs    []string
-	Badger              configurationtypes.CacheProvider
-	CDN                 configurationtypes.CDN
-	DefaultCacheControl string
-	Distributed         bool
-	Headers             []string
-	Olric               configurationtypes.CacheProvider
-	Regex               configurationtypes.Regex
-	TTL                 configurationtypes.Duration
-	Stale               configurationtypes.Duration
+	AllowedHTTPVerbs    []string                         `json:"allowed_http_verbs"`
```

Comment:
All of these will need godoc comments for the JSON config on the Caddy website to work correctly.